cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -384,7 +386,39 @@ def test_vm_reconfig_add_remove_network_adapters(adapters_type):
             4. Check the changes in VM reconfiguration page
             5. Remove the Adapters
     """
-    pass
+    orig_config = full_vm.configuration.copy()
+
+    # Create new configuration with new network adapter
+    new_config = orig_config.copy()
+    new_config.add_nw_adapter(
+        f"Network adapter {orig_config.num_nw_adapters + 1}", vlan=adapters_type
+    )
+    add_adapter_request = full_vm.reconfigure(new_config)
+    add_adapter_request.wait_for_request(method="ui")
+    request.addfinalizer(add_adapter_request.remove_request)
```

Comment:
Do you also want to remove the new config in finalizer ?

Reply:
discuses offline as its copy of `org_conf`

@sshveta added changes please take a look